### PR TITLE
Add documentations for Live Event and Callback

### PR DESCRIPTION
**Pull request by huynl-96**
## Related Tickets

- https://dev.framgia.com/issues/225580

## What's this PR do ?

- [x] Add documentations for live event
- [x] Add documentations for callback
- [x] Change getCode to getStatusCode

## Library
*(List maven plugins, library third party add new include version)*

## Impacted Areas in Application
*(List features, api, models or services that this PR will affect)*

## Checklist

- [x] It was tested in local success?
- [x] Fill link PR into ticket and the opposite
- [x] Note reason, scope of influence, solution into ticket
- [x] Validate UI/Model/API

## Deploy Notes
*(List commands, environment variables need configurations after deploy)*

## Notes
*(Other notes)*

### CODE REVIEW

**Review by ductb-0820 — APPROVED** [review]

File: doc/CALLBACK.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+  },
+  "method": "POST",
+  "status": 1,
+  "createdAt": "2018-06-23T01:27:08.000Z",
+  "updatedAt": "2018-06-23T01:27:08.000Z"
+}
+```
+
+## Update a callback
+Setup a callback to your server when an entity is completed for upload or public.
```
Comment (ductb-0820) [inline doc/CALLBACK.md:82]: Update callback chứ nhĩ? Docs Giống thằng create vậy

File: doc/LIVE.md [hunk 2]
```diff
@@ -0,0 +1,397 @@
...
+    "createdAt": "2018-12-13T17:30:42.000Z",
+    "updatedAt": "2018-12-13T17:30:42.000Z",
+    "entityName": "WATCH: SpaceX to Launch Falcon 9 Rocket #Spaceflight CRS16 @1:16pm EST"
+  }
+]
+```
+
+## Delete a record file
+Delete a recorded file.
+See details [here](https://docs.uiza.io/#stop-a-live-feed).
```
Comment (TuBaDuc) [inline doc/LIVE.md]: https://docs.uiza.io/?shell#delete-a-record-file

File: doc/LIVE.md [hunk 3]
```diff
@@ -0,0 +1,397 @@
...
+Example Response
+```java
+{
+  "id": "8b83886e-9cc3-4eab-9258-ebb16c0c73de",
+}
+```
+
+## Delete a record file
+Delete a recorded file.
+See details [here](https://docs.uiza.io/#stop-a-live-feed).
```
Comment (ductb-0820) [inline doc/LIVE.md]: sai link

File: doc/LIVE.md [hunk 4]
```diff
@@ -0,0 +1,397 @@
...
+    "extraInfo": null,
+    "endpointConfig": "s3-uiza-dvr",
+    "createdAt": "2018-12-13T17:30:42.000Z",
+    "updatedAt": "2018-12-13T17:30:42.000Z",
+    "entityName": "WATCH: SpaceX to Launch Falcon 9 Rocket #Spaceflight CRS16 @1:16pm EST"
+  }
+]
+```
+
+## Delete a record file
```
Comment (ductb-0820) [inline doc/LIVE.md:311]: sai title, sai ví dụ code

File: doc/CALLBACK.md (hunk 1 above)
Comment (huynl-96) [reply doc/CALLBACK.md:82]: Cái này em thấy trên docs.uiza nó để giống nhau nên mình cũng để tạm như vậy luôn ạ

File: doc/LIVE.md (hunk 4 above)
Comment (huynl-96) [reply doc/LIVE.md:311]: Dạ để em sửa lại

**Review by ductb-0820 — APPROVED** [review]

**Pull request merged**